graph/schema.graphqls

```diff
@@ -153,6 +167,17 @@ type SearchResult {
     For example, if searching for deployments, this will return the related pod resources.
     """
     related: [SearchRelatedResult]
+    """
+    Indicates if there are more results available after the current page.  
+    Calculated using: (offset + limit) < count
+    """
+    hasNextPage: Boolean
+    """
+    Indicates if there are results available before the current page.  
+    Calculated using: offset > 0 AND count > 0  
+    Returns false when there are no results (count = 0), even if offset is set.
+    """
+    hasPreviousPage: Boolean
```

Comment:
The client has all the data to calculate these.  The added fields are only for convenience. I'm worried the implementation is adding more complexity than a simple formula in the client.